woocommerce/google-listings-and-ads
# Streamline onboarding: Set the default value of the tax rate to destination-based

### Changes proposed in this Pull Request:

The Tax Rate (the UI will be removed later; see #2490), shown on the second screen when a US country is selected, must default to destination.

Closes  #2491.

Set the default value in the scheme property to `destination`. After the change, `/wp-json/wc/gla/mc/settings` does show default `tax_rate` as `destination`, in comparison to early `null.  This was also reflected in Tax Rate UI.
 See the before and after change screenshot of the setting endpoint. 

### Screenshots:
**Before**: you can see `tax_rate` is `null`
![before](https://github.com/user-attachments/assets/4041fc82-95ed-4ced-8654-22e82479355b)

**After**: you can see `tax_rate` is `destination`
![after](https://github.com/user-attachments/assets/2016146d-feda-448b-bf14-62bc40211ea8)





### Detailed test instructions:


1. Move to onboarding second screen. 
2. While on the second page, clear the option `npm run wp-env run cli wp option delete gla_merchant_center` to ensure it was not previously saved on a previous test.
3. Reload the 2nd screen, and under location select North American. You should see UI for `Tax rate (required for U.S. only)` below. 
4. Check if the default option is `My store uses destination-based tax rates.`.
5. Check db. `npm run wp-env run cli wp option get gla_merchant_center`. Note to QA:- You need to check this again once the UI removal is complete (#2490)


### Additional details:


### Changelog entry

> Tweak - Set the default value of the tax rate to destination-based during onboarding.
